SPECS/python-requests-oauthlib/python-requests-oauthlib.spec

```diff
@@ -9,7 +9,7 @@ Distribution:   Mariner
 
 Name:               python-requests-oauthlib
 Version:            1.2.0
-Release:            7%{?dist}
+Release:            8%{?dist}
```

Comment:
Same comment here regarding a clean-up during additions to the core specs. We should definitely remove any Python 2 parts, since we're never going to use them. Please refer to the previous comment for more details.